app/clients/hedera/mirror-client.go

```diff
@@ -68,14 +69,14 @@ func (c HederaMirrorClient) GetStateProof(transactionID string) ([]byte, error)
 	return readResponseBody(response)
 }
 
-func (c HederaMirrorClient) get(query string) (*http.Response, error) {
+func (c HederaMirrorClient) Get(query string) (*http.Response, error) {
 	return c.httpClient.Get(query)
 }
 
-func (c HederaMirrorClient) getTransactionsByQuery(query string) (*transaction.HederaTransactions, error) {
+func (c HederaMirrorClient) GetTransactionsByQuery(query string) (*transaction.HederaTransactions, error) {
```

Comment:
No need to make this public, you mock only the public method

Reply:
Fixed!